routes/utils.js

```diff
@@ -65,10 +171,29 @@ function updateCurrentProjectFiles(config, token, session, project, callback) {
   });
 }
 
+function getRemixedProjectFiles(config, projectId, callback) {
+  var publishURL = config.publishURL + "/publishedProjects/" + projectId + "/publishedFiles";
+
+  request.get({ uri: publishURL }, function(err, response, body) {
+    if(err) {
+      console.error("Failed to send request to " + publishURL + " with: ", err);
+      callback(null, 500);
```

Comment:
Whatever is passed as `err` in the first parameter will be propagated to the client, we don't want that for request errors, hence the console log.